composer.json

```diff
@@ -46,6 +46,9 @@
         }
     },
 
+    "minimum-stability": "dev",
+    "prefer-stable": true,
```

Comment:
-1 for this. This means there is nothing ensuring that Behat can be installed in ``minimum-stability: stable``.

You should change the minimum-stability in Travis jobs needing it instead (see how I did it in Assetic)